### PR TITLE
Fix style not passed as Card props

### PR DESCRIPTION
**Pull request by josepmartins**
## Summary

Fixes `style` not being passed as {...props} in the `Card` component

## Supporting resources (related issues, external links, etc):

- Slack thread: https://github.slack.com/archives/GACAW0NPM/p1695305993823789?thread_ts=1695302891.660489&cid=GACAW0NPM

## Contributor checklist:

- [x] All new and existing CI checks pass
- [x] Tests prove that the feature works and covers both happy and unhappy paths
- [x] Any drop in coverage, breaking changes or regressions have been documented above
- [x] New visual snapshots have been generated / updated for any UI changes
- [ ] All developer debugging and non-functional logging has been removed
- [ ] Related issues have been referenced in the PR description

## Reviewer checklist:

- [ ] Check that pull request and proposed changes adhere to our [contribution guidelines](../../CONTRIBUTING.md) and [code of conduct](../../CODE_OF_CONDUCT.md)
- [ ] Check that tests prove the feature works and covers both happy and unhappy paths
- [ ] Check that there aren't other open Pull Requests for the same update/change

## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

Screenshot 2023-09-22 at 09 24 49

 </td>
<td valign="top">

Screenshot 2023-09-22 at 09 24 12

</td>
</tr>
</table>

### CODE REVIEW

File: packages/react/src/Avatar/Avatar.visual.spec.ts-snapshots/Visual-Comparison-Avatar-Avatar-Default-1-linux.png
Comment (josepmartins) [inline packages/react/src/Avatar/Avatar.visual.spec.ts-snapshots/Visual-Comparison-Avatar-Avatar-Default-1-linux.png:1]: @rezrah any idea why this avatar snapshot is still appearing? 🤣

Comment (rezrah) [reply packages/react/src/Avatar/Avatar.visual.spec.ts-snapshots/Visual-Comparison-Avatar-Avatar-Default-1-linux.png:1]: 🤷 no idea... It is spooky season 👻 

It looks like the snapshot is being taken too early, before the image loads... We can add a manual delay in for it in a separate PR.

**Review by rezrah — APPROVED** [review]
LGTM...

FYI this will need a changeset. Recommend `patch`. Can this be done before merging please @josepmartins

**Review by auareyou — APPROVED** [review]
Looks good! Thanks @rezrah for helping me out.

**Pull request merged**